detd/devices/intel_i225.py

```diff
@@ -85,3 +86,22 @@ def supports_schedule(self, schedule):
         # FIXME: check additional constraints, like maximum cycle time
 
         return True
+    
+    def device_hints(self):
+        # Check the capabilities and set the Hints
+        preemption = False
+        launch_time_control = False
+        TxSelectionOffload = True
+        datapath = DataPath.AF_PACKET
+        txcapability = TxCapability.none
+
+        if Capability.Qbv in self.capabilities:
```

Comment:
By **default**, if Qbv HW is supported just set preemption and LTC as False. No need to check further. That would be a sane default.